### PR TITLE
add dogs, bni to the chain list

### PR DESCRIPTION
**Pull request by solidityx**

### CODE REVIEW

File: _data/chains/eip155-1117.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  },
+  "infoURL": "https://dogcoin.network",
+  "shortName": "DOGSm",
+  "chainId": 1117,
+  "networkId": 1117,
+  "explorers": [
+    {
+      "name": "Dogcoin",
+      "url": "https://mainnet.dogcoin.network",
+      "standard": "EIP3091"
```
Comment (ligi) [inline _data/chains/eip155-1117.json:20]: explorer is not reachable

**Review by ligi — CHANGES_REQUESTED** [review]
also why 2 jsons for same icons?

**Comment by solidityx** [comment]
> also why 2 jsons for same icons?

One for mainnet and second for testnet, but I'll remove and keep just one

**Review by ligi — APPROVED** [review]

**Review by ligi — APPROVED** [review]

**Comment by dicv2** [comment]
ARB

**Pull request merged**